src/Workspaces/Core/Portable/Utilities/Operations/ProgressInfo.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.Utilities
+{
+    /// <summary>
+    /// Represents an update of a progress.
+    /// </summary>
+    public readonly struct ProgressInfo
```

Comment:
part of the public api.  matches ProgressInfo from Microsoft.VisualStudio.Utilities.